lib/pure/strutils.nim

```diff
@@ -14,6 +14,7 @@
 ## <backends.html#the-javascript-target>`_.
 
 import parseutils
+import math
```

Comment:
Please list here what's actually used from ``math`` (``from math import foo, bar, baz``). The ``math`` dependency is not good, but I don't have a better alternative either so let's be a bit more verbose.